host/CMakeLists.txt

```
@@ -1,6 +1,6 @@
 #top dir cmake project for libhackrf + tools
 
-cmake_minimum_required(VERSION 2.8.12)
+cmake_minimum_required(VERSION 3.16)
```

```suggestion
cmake_minimum_required(VERSION 2.8...4.0)
```

If upstream wants to preserve compatibility this is how. Although if it's not tested with the lower end, there can be incompatible changes creeping in.

On the other hand for the users with very old CMake, wouldn't they also be using a very old version of this project? If they are building from source it is also trivial to get a modern cmake:
```console
$ pip install cmake
```